src/frontend/screens/Settings/ProjectSettings/YourTeam/ReviewAndInvite/index.tsx

```diff
@@ -40,8 +40,13 @@ export const ReviewAndInvite: NativeNavigationComponent<'ReviewAndInvite'> = ({
           navigation.navigate('InviteAccepted', {...route.params});
           return;
         }
+
+        if (val === 'REJECT') {
+          navigation.navigate('InviteDeclined', {...route.params});
```

Comment:
is object spread here necessary?

```suggestion
          navigation.navigate('InviteDeclined', route.params);
```

Reply:
https://github.com/digidem/comapeo-mobile/pull/262/commits/80658c7bcd722a37ecf37660e8acb0d5a72d8269